### DISCUSSION TITLE
User session

### DISCUSSION THREAD

**Discussion by mtophir** [maxpert/marmot#56] [Q&A] [answered]
Not sure if anyone has experienced this issue in testing. I have 2 instances running and when I logged in to the first instance, my login session on the second instance is gone. I had to login again on the second instance, this lead to my login session on the first instance getting wiped. 
I think in the Keystone demo, the user has the same issue.
Thanks.

**Comment by maxpert** [comment]
Is this issue in Pocketbase?

**Comment by mtophir** [comment]
No idea how Pocketbase works, I did not see the user in the demo doing any
login using a browser.

In my case, I stored user data in sqlite. After logging in, I stored the
userid in a web session. For some reason, after each update to a table
which triggered a replication, the session is wiped.

**Reply by maxpert** [reply]
I just debugged the tables, turns out there is a token setting in `_params` table. You need to make sure these params match up. Hopefully PB is not loading anything into memory, and it reads updated tokens from table.

**Reply by maxpert** [reply]
I just verified with same DB deployed on two instances it works.

**Reply by maxpert** [reply, marked as answer]
https://github.com/pocketbase/pocketbase/discussions/2898 I've posted a working example
